saos-webapp/src/main/resources/message/common.properties

```diff
@@ -186,6 +187,13 @@ judgmentDetails.corrections.value.DECISION = Postanowienie
 judgmentDetails.corrections.value.REGULATION = Zarządzenie
 judgmentDetails.corrections.value.OTHER = Inny
 
+/* */
```

Comment:
Empty comment. You could add here the same as in `common_pl.properties`. That is `/* Analysis */`.